mattermost/mattermost-plugin-jira
# [MM-803]: Added a slash command to set a default jira instance in case of multiple instances

### Summary
Added a slash command to set a default instance in case of multiple Jira instances
`/jira instance default <jiraURL>`

### Screenshot
![Screenshot from 2024-07-31 15-14-53](https://github.com/user-attachments/assets/17f512f1-78e9-418d-832b-bcd30e71e4db)
![Screenshot from 2024-07-31 15-15-11](https://github.com/user-attachments/assets/477b670b-1182-4446-bb26-6763619f5754)
![Screenshot from 2024-07-31 15-15-29](https://github.com/user-attachments/assets/3ebcf50f-60d8-4b55-8305-e0286300c2cc)
![Screenshot from 2024-07-31 15-21-52](https://github.com/user-attachments/assets/fce116c8-ac39-45d6-8d9e-f26178593926)

### Issue Link
 Fixes https://github.com/mattermost/mattermost-plugin-jira/issues/803
 
 ### What to test
 - Connect Mattermost to 2 JIra instances but don't create any issue or subscription
 - Run `/jira instance setting list`, you must receive an error as mentioned in https://github.com/mattermost/mattermost-plugin-jira/issues/803
 - Run `/jira instance default <jiraURL>`
- Run `/jira instance setting list`, you must receive not receive any error and the command should respond as expected
